app/lib/actions/candlepin/environment/add_content_to_environment.rb

```diff
@@ -0,0 +1,18 @@
+module Actions
+  module Candlepin
+    module Environment
+      class AddContentToEnvironment < Candlepin::Abstract
+        input_format do
+          param :view_env_cp_id
+          param :content_id
+        end
+
+        def run
+          ::Katello::Resources::Candlepin::Environment.add_content(input[:view_env_cp_id], [input[:content_id]])
```

Comment:
The existing SetContent action puts the response in output[:add_response] probably for debugging purposes. That could be useful to do here, also for the consistency between similar actions.